jdbc/src/test/java/org/apache/iotdb/jdbc/IoTDBQueryResultSetTest.java

```diff
@@ -129,6 +129,9 @@ public void before() throws Exception {
 
     statement = new IoTDBStatement(connection, client, sessionId, zoneID);
 
+    TSQueryDataSet tsQueryDataSet = FakedFirstFetchResult();
+    execResp.queryDataSet = tsQueryDataSet;
```

Comment:
Local variable 'tsQueryDataSet' is redundant, you can directly combine the two statements.

Reply:
fixed